plugins/plugin-languageserver/che-plugin-languageserver-ide/pom.xml

```diff
@@ -149,7 +149,7 @@
                 <version>${project.version}</version>
                 <executions>
                     <execution>
-                        <phase>process-sources</phase>
+                        <phase>generate-sources</phase>
```

Comment:
The maven generator plugin is using compiled dependencies to generate the dto classes. Since no dto's are generated from classes in the project itself, this should be fine. In the core.api.languageserver project, there used to be a precompile step in order to be able to generate from classes in the project itself.